tests/tasks/check_firewall_selinux.yml

```diff
@@ -0,0 +1,50 @@
+# SPDX-License-Identifier: MIT
+---
+- name: Get firewall service
+  command: firewall-cmd --list-services
+  register: _result
+  changed_when: false
+
+- name: Check firewall service status when vpn_manage_firewall is true
+  assert:
+    that:
+      - "'ipsec' in _result.stdout"
+  when: vpn_manage_firewall | d(true) | bool
+
+- name: Check firewall service status when vpn_manage_firewall is false
+  assert:
+    that:
+      - "'ipsec' not in _result.stdout"
+  when: not vpn_manage_firewall | d(true) | bool
+
+- name: Get associated selinux ports
+  shell: |-
+    set -euo pipefail
+    firewall-cmd --info-service=ipsec | \
+      egrep " +ports: +" | sed -e "s/ *ports: //"
+  register: __ports
+  changed_when: false
+
+- name: Check associated selinux ports when vpn_manage_selinux is true
+  shell: |-
+    set -euo pipefail
+    sudo semanage port --list -C | grep ipsecnat_port_t | \
+      grep "{{ pair[0] }}" | grep "{{ pair[1] }}"
+  changed_when: false
+  vars:
+    pair: "{{ item.split('/') }}"
+  loop: "{{ __ports.stdout.split(' ') }}"
+  when: vpn_manage_selinux | d(true) | bool
+
+- name: Check associated selinux ports when vpn_manage_selinux is false
+  shell: |-
+    set -euo pipefail
+    sudo semanage port --list -C | grep ipsecnat_port_t | \
```

Comment:
Is it a common practice to call `sudo` manually rather than using [`become`](https://docs.ansible.com/ansible/latest/user_guide/become.html)?